src/c_kzg_4844.c

```diff
@@ -990,14 +990,25 @@ C_KZG_RET blob_to_kzg_commitment(
     return C_KZG_OK;
 }
 
+/**
+ * Struct holding the argument to verify_kzg_proof_impl.
+ * Having a struct holding these allows some code simpliciations
+ *
+ * An instance of this struct models a claim (to be verified) that:
+ * proof_g1 is a proof for p(evaluation_point) == y_fr,
```

Comment:
If we want to change the name of `evaluation_point` to `z_fr` as @asn-d6 suggested, this has to be updated as well.